Makefile

```diff
@@ -112,7 +112,7 @@ BUNDLE_IMG ?= $(IMAGE_TAG_BASE)-bundle:v$(VERSION)
 # Image URL to use all building/pushing image targets
 IMG ?= quay.io/konveyor/oadp-operator:latest
 # Produce CRDs that work back to Kubernetes 1.11 (no version conversion)
-CRD_OPTIONS ?= "crd:trivialVersions=true,preserveUnknownFields=false"
+CRD_OPTIONS ?= "crd:trivialVersions=true,preserveUnknownFields=false,allowDangerousTypes=true"
```

Comment:
I am not so sure that we should add `allowDangerousTypes=true`

Reply:
to have conversation in one place, will move this comment to here https://github.com/openshift/oadp-operator/pull/1395#discussion_r1610114100